requirements.txt

```diff
@@ -1,7 +1,7 @@
-matplotlib==3.3.4
+matplotlib==3.7.0
 numpy==1.23.0
-pandas==1.1.5
-scikit-learn==0.24.2
+pandas==2.0.0
```

Comment:
Should we bump up to the [current version](https://pandas.pydata.org/docs/whatsnew/v2.1.1.html)?